contrib/scripts/export-creds.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env sh
```

Comment:
You're specifying a POSIX shell here, but several of the features you're using below aren't available in the POSIX shell. I ran [shellcheck](https://github.com/koalaman/shellcheck) over this script and I'll add the POSIX errors it reports to the lines that triggered them.